modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -408,36 +408,36 @@ Status description:: error: syntax error or access rule violation - invalid symb
 
 === 42I25
 
-Status description:: error: syntax error or access rule violation - invalid use of CALL IN TRANSACTIONS. 'CALL { ... }` IN TRANSACTIONS' is not supported after a write clause.
+Status description:: error: syntax error or access rule violation - invalid use of `CALL IN TRANSACTIONS`. 'CALL { ... } IN TRANSACTIONS' is not supported after a write clause.
 
 === 42I29
 
-Status description:: error: syntax error or access rule violation - invalid use of IS. The IS keyword cannot be used together with multiple labels in `{ $input }`. Rewrite the expression as `{ $replacement }`.
+Status description:: error: syntax error or access rule violation - invalid use of `IS`. The `IS` keyword cannot be used together with multiple labels in `{ $input }`. Rewrite the expression as `{ $replacement }`.
 
 === 42I31
 
-Status description:: error: syntax error or access rule violation - invalid use of MATCH. MATCH ...' cannot directly follow an 'OPTIONAL MATCH ...'. Use a WITH clause between them.
+Status description:: error: syntax error or access rule violation - invalid use of `MATCH`. 'MATCH ...' cannot directly follow an 'OPTIONAL MATCH ...'. Use a `WITH` clause between them.
 
 
 === 42I37
 
-Status description:: error: syntax error or access rule violation - invalid use of `RETURN *`. `RETURN *'` is not allowed when there are no variables in scope.
+Status description:: error: syntax error or access rule violation - invalid use of `RETURN *`. 'RETURN *' is not allowed when there are no variables in scope.
 
 === 42I38
 
-Status description:: error: syntax error or access rule violation - invalid use of `RETURN`. `RETURN ...'` can only be used at the end of a query or subquery.
+Status description:: error: syntax error or access rule violation - invalid use of `RETURN`. 'RETURN ...' can only be used at the end of a query or subquery.
 
 === 42I40
 
-Status description:: error: syntax error or access rule violation - invalid use of `UNION and UNION ALL`. `UNION` and `UNION ALL` cannot be combined.
+Status description:: error: syntax error or access rule violation - invalid use of `UNION` and `UNION ALL`. `UNION` and `UNION ALL` cannot be combined.
 
 === 42I47
 
 Status description:: error: syntax error or access rule violation - parser error. Parser Error: `{ $msg }`.
 
 === 42I50
 
-Status description:: error: syntax error or access rule violation - token name too long. Invalid input `{ $input }`... A `{ $tokenType }` name cannot be longer than `{ $maxTokenLength }`.
+Status description:: error: syntax error or access rule violation - token name too long. Invalid input `{ $input }..`. A `{ $tokenType }` name cannot be longer than `{ $maxTokenLength }`.
```

Comment:
I am not sure about putting the backticks in between the dots on this one. I guess the dots are supposed to be ... like in etc. but also they are in the end of a sentence. I leave it up to you, how you think it is best to format

Reply:
Yeah I was thinking the same way, let's keep it like this and change it later if someone would complain (which I doubt)